pkg/diagnostics/systemd/systemd.go

```diff
@@ -108,7 +108,7 @@ log message:
   component. Check pod logs and recreate it with the correct CA cert.
   Routers and registries won't work properly with the wrong CA.
 * If it is from a node IP, the client is likely a node. Check the
-  openshift-node logs and reconfigure with the correct CA cert.
+  atomic-openshift-node logs and reconfigure with the correct CA cert.
```

Comment:
I'm torn as to whether or not we should make this messaging more general and say 'the node logs' or perhaps 'origin-node or atomic-openshift-node logs' ? Not sure it's worth changing from what you've proposed.